### ISSUE TITLE
CreateReducer.js uses "startsWith" which is unsupported in IE11

### ISSUE THREAD

**Issue by soavie** [erikras/redux-form#4126]
### Are you submitting a **bug report** or a **feature request**?

bug report

### What is the current behavior?

Unable to input text into Redux Form Field in IE11. Results in "TypeError: Object doesn't support property or method 'startsWith'" 

### What is the expected behavior?
It should be possible to input text without throwing any errors in IE11.

### What's your environment?

redux-form 7.4.2
IE11

### Other information

Line causing the error:
https://github.com/erikras/redux-form/blob/v7.4.2/src/createReducer.js#L47
```
  if (path.startsWith('values')) {
```
Commit that introduced the error:
https://github.com/erikras/redux-form/commit/a97606fee0ed365903fff80cc93956fe2313a3ce

`String.prototype.startsWith()` browser support:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

We've been able to work around this by using [core-js](https://github.com/zloirock/core-js) to polyfill the feature.

**Comment by gustavohenke** [comment]
Thanks for the heads up. That seems to have been introduced in v7.4.0, and should be an easy fix, if you would like to go for it 🙂

**Comment by chrisjlee** [comment]
Any preference how this is done? 

is this polyfill sufficient?

```
if (!String.prototype.startsWith) {
	String.prototype.startsWith = function(search, pos) {
		return this.substr(!pos || pos < 0 ? 0 : +pos, search.length) === search;
	};
}
```

**Comment by gustavohenke** [comment]
Adding polyfills to Redux Form wouldn't be a good idea, but if you mean using its implementation, then yeah, that's an option.

An even better one would be to use a regular expression, like `/^values/.test(path)`, as then others don't need to guess where the magic number of the `.substr()` call is coming from.

**Comment by chrisjlee** [comment]
Yes implementation-wise. Thank you @gustavohenke !

**Comment by gustavohenke** [comment]
PR #4128 with the fix has been merged. Watch out for new releases!

**Comment by ldthorne** [comment]
Hey @gustavohenke @erikras , do you know when there will be a new release with this included? I'd like to use this fix as I'm hitting the same issue, but haven't been able to because there's no release for it yet. Thanks!

**Comment by razagill** [comment]
@erikras @gustavohenke any update on this?

**Comment by DonWasyl** [comment]
I also waiting for this fix :(

**Comment by gustavohenke** [comment]
I don't control the release cycle.

**Comment by leocristofani** [comment]
This issue is marked as `closed`, but it persists because https://github.com/erikras/redux-form/pull/4128 has been merged into master, but not released yet. Any idea when this is going to be released? Thanks very much!

**Comment by davidgtu** [comment]
I know this was merged in, but any updates on this?

**Comment by davidgtu** [comment]
@soavie I'm having this issue as well. how did you get around this using core-js?

**Comment by abumalick** [comment]
`import 'core-js/modules/es6.string.starts-with'; // redux-form`
in the beginning of your app

**Comment by erikras** [comment]
Published fix in [`v8.0.0`](https://github.com/erikras/redux-form/releases/tag/v8.0.0).
